src/GraphQL/Execution/Executor.php

```diff
@@ -113,10 +113,10 @@ class Executor implements ExecutorImplementation {
    * @param \GraphQL\Type\Schema $schema
    * @param \GraphQL\Language\AST\DocumentNode $document
    * @param \Drupal\graphql\GraphQL\Execution\ResolveContext $context
-   * @param $root
-   * @param $variables
-   * @param $operation
-   * @param $resolver
+   * @param string $root
+   * @param string $variables
```

Comment:
should be an array according to the doc comment on the class variable where this is assigned to. Please check those